integration-tests/cloudflare-workers/worker/src/index.ts

```diff
@@ -12,26 +12,41 @@
  */
 
 export interface Env {
-	OPENAI_API_KEY: string;
+  OPENAI_API_KEY: string;
 }
 
+import {
+  Agent,
+  BatchTraceProcessor,
+  ConsoleSpanExporter,
+  getGlobalTraceProvider,
+  run,
+  setDefaultOpenAIKey,
+  setTraceProcessors,
+  startTraceExportLoop,
+} from '@openai/agents';
+
 export default {
-	async fetch(request, env, ctx): Promise<Response> {
-		try {
-			const { Agent, run, setDefaultOpenAIKey, setTracingDisabled } = await import('@openai/agents');
+  async fetch(request, env, ctx): Promise<Response> {
+    try {
+      setDefaultOpenAIKey(env.OPENAI_API_KEY!);
+      setTraceProcessors([new BatchTraceProcessor(new ConsoleSpanExporter())]);
+      startTraceExportLoop();
+
+      const agent = new Agent({
+        name: 'Test Agent',
+        instructions:
+          'You will always only respond with "Hello there!". Not more not less.',
+      });
+      const result = await run(agent, 'Hey there!');
 
-			setDefaultOpenAIKey(env.OPENAI_API_KEY!);
-			setTracingDisabled(true);
+      // make sure you shut down tracing before exiting to flush any remaining traces
```

Comment:
a problem here is that a worker isolate might be serving multiple requests, so this might shutdown before other requests are complete. I'll sit down this evening after dinner and try to come up with something better (probably using ALS)

Reply:
another issue is that if this returns a websocket, then the response is returned early, and any traces collected during the websocket's lifetime won't be flushed